### PR TITLE
feat: Start using the new item model for most features

### PR DESCRIPTION
**Pull request by magicus**
Here is most of the remaining work for the item rewrite. The only thing still left behinds are tooltips, which is another level of messiness, and will take some more work to sort out fully. :-(

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/user/LobbyUptimeFeature.java [hunk 1]
```diff
@@ -7,11 +7,43 @@
...
 
 public class LobbyUptimeFeature extends UserFeature {
     @Override
     public List<Model> getModelDependencies() {
-        return List.of(Models.ServerList, Models.ServerItemStack);
+        return List.of(Models.ServerList);
+    }
+
+    @SubscribeEvent
+    public void onTooltipPre(ItemTooltipRenderEvent.Pre event) {
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/user/LobbyUptimeFeature.java:27]: Are all item stack tooltips injected like this? I am sure this does not look great in allocations if we do this for gear items

**Review by kristofbolyai — APPROVED** [review]
Looks good

File: common/src/main/java/com/wynntils/features/user/LobbyUptimeFeature.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/features/user/LobbyUptimeFeature.java:27]: Tooltips are not handled in this PR. Having proper caching of tooltips is one reason why they are more tricky.

File: common/src/main/resources/assets/wynntils/lang/en_us.json [hunk 2]
```diff
@@ -159,6 +159,11 @@
   "feature.wynntils.emeraldPouchHotkey.multipleFilled": "You have multiple filled emerald pouches in your inventory.",
   "feature.wynntils.emeraldPouchHotkey.name": "Emerald Pouch Hotkey",
   "feature.wynntils.emeraldPouchHotkey.noPouch": "You do not have an emerald pouch in your inventory.",
+  "feature.wynntils.extendedItemCount.hotbarTextOverlayEnabled.description": "Should we draw extended item counts for items in the hotbar?",
+  "feature.wynntils.extendedItemCount.hotbarTextOverlayEnabled.name": "Exteded Hotbar Item Counts",
+  "feature.wynntils.extendedItemCount.inventoryTextOverlayEnabled.description": "Should we draw extended item counts for items in the inventory?",
+  "feature.wynntils.extendedItemCount.inventoryTextOverlayEnabled.name": "Exteded Inventory Item Counts",
```
Comment (Jimmy-sheep) [inline common/src/main/resources/assets/wynntils/lang/en_us.json:165]: `Exteded` is this typo?

Comment (magicus) [reply common/src/main/resources/assets/wynntils/lang/en_us.json:165]: yeah, thanks!

**Pull request merged**